frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss

```diff
@@ -143,6 +143,10 @@
   align-items: center;
   display: flex;
   overflow: hidden;
+
+  .pf-v5-c-icon {
```

Comment:
Is it possible to add a custom class so we we're not introducing a PF version dependency?

Reply:
Updated the markup to wrap with `<Icon>` component and apply custom style rule.